x-pack/plugins/uptime/public/components/overview/filter_group/filter_popover.test.tsx

```diff
@@ -77,17 +77,23 @@ describe('FilterPopover component', () => {
 
       fireEvent.click(uptimeFilterButton);
 
-      const generateLabelText = (item: string) => `Filter by ${props.title} ${item}.`;
+      const generateLabelText = (item: string, isSelected: boolean) => {
+        if (selectedPropsItems.find((i) => i === item)) {
+          return `${isSelected ? 'Remove filter' : 'Filter'} by ${props.title} ${item}.`;
```

Comment:
Any reason this doesn't use the content helpers you created below? Seems like the content is the same.

Reply:
I simplified the test to not have that inline function, and am using the translation format functions (assuming that's what you meant).